src/main/java/org/fundacion/jala/converter/models/Asset.java

```diff
@@ -55,88 +56,99 @@ public Asset(final String newTitle, final String newPath, final String newChecks
     }
 
     /**
-     * Obtains Id of the project
-     * @return id of the project
+     * Obtains Id of the database.
```

Comment:
It should be id from the database, not of.

Reply:
ok it was solved